lib/sort.dart

```diff
@@ -156,8 +154,10 @@ ImportSortData sortImports(
   final sortedFile = sortedLines.join('\n');
   final original = lines.join('\n') + '\n';
   if (exitIfChanged && original != sortedFile) {
-    stdout.write('\n┗━━🚨 ');
-    stdout.write('Please run import sorter!'.bold().red());
+    if (filePath != null) {
+      stdout.writeln(
+          '\n┗━━🚨 File ${filePath} does not have its imports sorted.');
```

Comment:
Please bring back the red and bold display. It's most likely a problem with your terminal's display colors/system if the color isn't being displayed correctly. It works fine for me.

Reply:
I didn't customize anything - just a default Ubuntu terminal (test it out for yourself – it's available on GitHub Actions). I think that many people are using Ubuntu and may encounter this.

Reply:
Ok, we can cut it then